### ISSUE TITLE
Store adding grouper gives an undefined error

### ISSUE THREAD

**Issue by gryffindor14** [ahome-it/ahome-sencha-shared#3]
the function add grouper gives a "cannot use function add of undefined". The jso.groupers field is never initialized 

var jso = this.@com.ait.toolkit.core.client.JsObject::getJsObj()();
		jso.groupers
				.add(grouper.@com.ait.toolkit.core.client.JsObject::getJsObj()());

**Comment by eemi2010** [comment]
Thx for reporting.
Will have a look.

**Comment by gryffindor14** [comment]
There should only be 1 grouper per store. Currently there is also a bug where setting a grouper sets a sorter instead.

the functions getGroupers and setGroupers should be changed to use a singular grouper object.

Extjs and sencha touch stores both have getGrouper() and setGrouper(object) methods that can be used. There is no addGrouper function so that should also be removed or modified as well

**Comment by eemi2010** [comment]
Yeah Sencha changed store with Ext JS 5+ and Sencha Touch 2.4. We are working on  supporting the new API while keeping backward compatibility because we still need to support Ext JS 4+ and Sencha Touch till version 2.3.  Fixes for this should land soon.
